### ISSUE TITLE
button-merchant-ios fails to build on the latest Xcode beta 

### ISSUE THREAD

**Issue by glja021** [button/button-merchant-ios#118]
Hello button team 👋 
I would like to report a build error we're seeing when building against the latest Xcode 16 beta (`16A5211f`)

```
button-merchant-ios/Source/Extensions/UIApplicationExtensions.swift:33:1 Type 'UIApplication' does not conform to protocol 'UIApplicationType'
Candidate has non-matching type '(URL, [UIApplication.OpenExternalURLOptionsKey : Any], (@MainActor @Sendable (Bool) -> Void)?) -> Void' (UIKit.UIApplication)
```

**Comment by regularberry** [comment]
We're also seeing this in Xcode 16 Beta 5

**Comment by wessmith** [comment]
Thanks for reporting! This is resolved in [1.7.1](https://github.com/button/button-merchant-ios/releases/tag/1.7.1)

**Comment by stefanofrosoni** [comment]
I can still see this issue on Xcode 16 beta 6 .
